pkg/metrics/metrics.go

```diff
@@ -123,36 +83,6 @@ func ServePrometheus(useTLS bool, cert, key string) error {
 // registerMetrics registers marketplace prometheus metrics.
 func registerMetrics() error {
```

Comment:
Marketplace no longer registers _any_ metrics, so is there a reason to leave the metrics port open? compliance?

Reply:
I left it in there predicting we might want metrics for our default CatalogSources. Ripping them out now seems premature.